module_intros/diffraction.StaticStructureFactor.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>It's not necessary to put the formula here if it is included in the class docstrings.</p><p> </p><p>I think you meant <code>query_points</code>  and not <code>nquery</code>  ; likewise <code>N_total</code>  instead of <code>N_tot</code>. <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>